api/.mvpstudio/kube/runit.yml

```diff
@@ -0,0 +1,58 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: mvpstudio-web-api
+  namespace: stage-mvpstudio-web
+  labels:
+    app: mvpstudio-web-api
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: mvpstudio-web-api
+  template:
+    metadata:
+      labels:
+        app: mvpstudio-web-api
+    spec:
+      containers:
+      - name: api-server
+        image: mvpstudio/mvpstudio-web-api
```

Comment:
this image needs a tag, otherwise it will always pull the latest

Reply:
Will update this when I publish the container to dockerhub